index.html

```diff
@@ -5,11 +5,49 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://unpkg.com/aos@next/dist/aos.css" />
+    <link rel="stylesheet" href="style.css">
     <title>Document</title>
 </head>
 
+
 <body>
-    <h1>My heading</h1>
+    <header>
+        <h1>COLORADO LIFE</h1>
+        <figure><img src="images/god.jpg" alt="god's garden">
+        </figure>
+    </header>
+    <p>Morning</p>
+    <div class=morning>
+        <figure><img src="images/dog.jpg" alt="dog next to the bed"></figure>
+        <figcaption>When I woke up in the morning, the dog, sparky, was lie down on the floor next to the bed.
```

Comment:
Ideally, you want your `figcaption` to be inside of your `figure` tag, for example:
```html
<figure>
  <img>
  <figcaption>caption here</figcaption>
</figure>
```